apps/web/src/pages/ApplicantDashboardPage/ApplicantDashboardPage.tsx

```diff
@@ -32,8 +31,9 @@ export const ApplicantDashboardPage_Fragment = graphql(/* GraphQL */ `
     firstName
     lastName
     isGovEmployee
-    workEmail
-    isWorkEmailVerified
+    isVerifiedGovEmployee
+    hasPriorityEntitlement
```

Comment:
Is there a better way of typing helper functions to catch under-fetching like this?

Reply:
I was kind of thinking about this myself. I think if we can avoid using types like this:

https://github.com/GCTC-NTGC/gc-digital-talent/blob/7c209c78367b62fb84f41a9ae8f799f768fe5212/apps/web/src/validators/profile/governmentInformation.ts#L4-L17

We might be better off. The problem is, unless the schema marks them as non nullable we get something like this:

```tsx
type User = {
  hasPriorityEntitlement?: Maybe<string>;
}|
```

So it would never get caught if we don't fetch it because it is totally valid to be `undefined.` I think something like this would be more accurate:

```tsx
type HasAllEmptyFieldsArgs = {
  hasPriorityEntitlement: string | null;
}
```

This way, we are forced to pass it in.

Reply:
lol turns out I already noted this and created a ticket for it :laughing: 

https://github.com/GCTC-NTGC/gc-digital-talent/issues/12736